classes/class-newsmatchdonation-settings.php

```diff
@@ -0,0 +1,257 @@
+		);
+
+		register_setting( $this->settings_group, self::$options_prefix . 'default_sf_id', 'sanitize_text_field' );
+		add_settings_field(
+			self::$options_prefix . 'default_sf_id',
+			__( 'Default Salesforce campaign id', 'newsmatch' ),
+			array( $this, 'field_default_sf_id' ),
+			$this->settings_page,
+			$this->settings_section
+		);
+	}
+
+	/**
+	 * Settings section description
+	 */
+	public function settings_section_callback() {
+		echo '';
+	}
+
+	/**
+	 * Output the input field for the organization's name
+	 *
+	 * @param array $args Optional arguments passed to callbacks registered with add_settings_field.
+	 */
+	public function field_org_name( $args ) {
+		$option = self::$options_prefix . 'org_name';
+		$value = get_option( $option, '' );
+		echo sprintf(
+			'<input name="%1$s" id="%1$s" type="text" value="%2$s">',
+			esc_attr( $option ),
+			esc_attr( $value )
+		);
+	}
+
+	/**
+	 * Output the input field for the organization's org_id
+	 *
+	 * This is part of the donation form URL
+	 *
+	 * @param array $args Optional arguments passed to callbacks registered with add_settings_field.
+	 */
+	public function field_org_id( $args ) {
+		$option = self::$options_prefix . 'org_id';
+		$value = get_option( $option, '' );
+		echo sprintf(
+			'<input name="%1$s" id="%1$s" type="text" value="%2$s">',
+			esc_attr( $option ),
+			esc_attr( $value )
+		);
+	}
+
+	/**
+	 * Output the input field for the live donation form URL
+	 *
+	 * @param array $args Optional arguments passed to callbacks registered with add_settings_field.
+	 */
+	public function field_url_live( $args ) {
+		$option = self::$options_prefix . 'url_live';
+		$value = get_option( $option, '' );
+		echo sprintf(
+			'<input name="%1$s" id="%1$s" type="text" value="%2$s">',
+			esc_attr( $option ),
+			esc_attr( $value )
+		);
+	}
+
+	/**
+	 * Output the input field for the staging/testing donation form URL
+	 *
+	 * @param array $args Optional arguments passed to callbacks registered with add_settings_field.
+	 */
+	public function field_url_staging( $args ) {
+		$option = self::$options_prefix . 'url_staging';
+		$value = get_option( $option, '' );
+		echo sprintf(
+			'<input name="%1$s" id="%1$s" type="text" value="%2$s">',
+			esc_attr( $option ),
+			esc_attr( $value )
+		);
+	}
+
+	/**
+	 * Ouput the radio button for toggling between 'live' and 'testing' urls
+	 *
+	 * @param array $args Optional arguments passed to callbacks registered with add_settings_field.
+	 */
+	public function field_url_toggle( $args ) {
+		$option = self::$options_prefix . 'url_toggle';
+		$value = get_option( $option, 'staging' );
+		if ( ! in_array( $value, array( 'staging', 'live' ), true ) ) {
+			$value = 'staging';
+		}
+
+		echo sprintf(
+			'<input name="%1$s" id="%1$s-staging" type="radio" value="staging" %2$s><label for="%1$s-staging">%4$s</label>
+			<input name="%1$s" id="%1$s-live" type="radio" value="live" %3$s><label for="%1$s-live">%5$s</label>',
```

Comment:
The UX on this doesn't currently match any existing core styles - checkout `/wp-admin/options-reading.php` for an example of the markup of radio buttons to match the style of existing core UX patterns